lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py

```diff
@@ -113,14 +113,30 @@ def get_body_from_alb_invocation(event):
     except:
         raise
 
+def read_from_ssm(key: str) -> str:
+    boto_client = boto3.client('ssm')
+    
+    resp = boto_client.get_parameter(
+        Name=f"{key}",
+        WithDecryption=True
+    )
+
+    logging.info(resp)
+
+    return resp["Parameter"]["Value"]
```

Comment:
You could make this more error resistant by catching KeyError is doing something like this:
```
return response.get("Parameter", {}).get("value", '')
```